ec2/spark_ec2.py

```diff
@@ -1007,6 +1022,14 @@ def real_main():
         print >> stderr, "ebs-vol-num cannot be greater than 8"
         sys.exit(1)
 
+    # Limit naming to avoid breaking things, as we rely on the repo
```

Comment:
Sorry, misread/understood. Will change and test. May as well keep the CLI check, but use it to  ensure there's no trailing / or .git.